_docs/concepts/policy-and-control/mixer-config.md

```diff
@@ -50,357 +47,222 @@ known as [*adapters*](./mixer.html#adapters).
 <figure><img src="./img/mixer-config/machine.svg" alt="Attribute Machine" title="Attribute Machine" />
 <figcaption>Attribute Machine</figcaption></figure>
 
-Mixer's configuration has two central responsibilities:
+Mixer's configuration has the following central responsibilities:
 
 - Describe which adapters are being used and how they operate.
-- Describe how to map request attributes into adapter parameters.
+- Describe how to map request attributes into adapter inputs.
+- Describe when a particular adapter is called with specific inputs.
 
-Configuration is expressed using a YAML format built around five core
-abstractions:
+Configuration is based on  *adapters* and *templates*.
+- **Adapter** encapsulate the logic necessary to interface Mixer with a specific infrastructure backend.
```

Comment:
Suggestion: Define what the infrastructure backend means, earlier in the doc, or just simply list out the common things that comprise this "infrastructure backend".